### PR TITLE
Only run tests with +noCMAForZC if CMK_USE_CMA is true

### PR DESCRIPTION
**Pull request by evan-charmworks**

### CODE REVIEW

**Comment by evan-charmworks** [comment]
@nitbhat Since your machine supports CMA, could you test that this patch functions correctly?

**Comment by nitbhat** [comment]
Yes, testing it now.

**Comment by nitbhat** [comment]
On my CMA enabled machine, I do see `checking "whether has support for shm transport using Cross Memory Attach"... "yes"` while building. However, `CMK_USE_CMA` is `0` in `conv-mach-opt.mak`, causing `+noCMAForZC` option tests to not run.

**Comment by nitbhat** [comment]
I tested it on `courage` (a lab machine). If you have access, you can test it on that.

**Comment by evan-charmworks** [comment]
Sorry about that, should be fixed now.

File: examples/charm++/zerocopy/entry_method_post_api/prereg/nodegroupTest/Makefile [hunk 1]
```diff
@@ -21,6 +21,7 @@ else
 	$(call run, +p4 ./nodegroupTest)
 	$(call run, +p4 ./nodegroupTest +noCMAForZC)
 endif
+endif
```
Comment (matthiasdiener) [inline examples/charm++/zerocopy/entry_method_post_api/prereg/nodegroupTest/Makefile]: this additional `endif` seems spurious.

**Pull request merged**